### PR TITLE
Bug 2141 - Add a project selector index page

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: make.js [hunk 1]
```diff
@@ -467,4 +468,11 @@ target.deploy = function(){
     exec( UGLIFY + ' --output ' + BUTTERED_POPCORN + ' ' + BUTTERED_POPCORN );
   }
   mv( BUTTERED_POPCORN, './dist/external/popcorn-js/popcorn.js' );
+
+  // Move everything into the public folder
+  cp('-R', 'public', DIST_DIR);
+  mv(['dist/css', 'dist/editors', 'dist/external', 'dist/resources', 'dist/src', 'dist/templates'], 'dist/public/');
+
+  // It's important to use the production config
+  echo('Run cornfield with `NODE_ENV=production node app.js`');
```
Comment (secretrobotron) [inline make.js]: Can you just clean up the spacing here? Trying to persist with Popcorn style.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brett-etherworks received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/b5342bd2140d631/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b5342bd2140d631/output.txt

Total script time: 0.91 mins

Server is running on http://pr902.butyr.org/templates/basic/
Run the Test Suite: http://pr902.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/4140b1a925dbec8/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/a9bbfa575ae1bb3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 2

Live output at: http://botio.butyr.org:8000/871a4f8d6c19fa2/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/4140b1a925dbec8/output.txt

Total script time: 2.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/a9bbfa575ae1bb3/output.txt

Total script time: 1.68 mins

Server is running on http://pr902.butyr.org/templates/basic/
Run the Test Suite: http://pr902.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/aed22ebd2e15871/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/871a4f8d6c19fa2/output.txt

Total script time: 1.66 mins

Server is running on http://pr902.butyr.org/templates/basic/
Run the Test Suite: http://pr902.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/aed22ebd2e15871/output.txt

Total script time: 1.87 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/ad1cd2d5b59f454/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/601cf491b08b6c2/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/ad1cd2d5b59f454/output.txt

Total script time: 2.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/601cf491b08b6c2/output.txt

Total script time: 0.01 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/c2c324a22d6cf15/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/3b311ab37a5754d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/c2c324a22d6cf15/output.txt

Total script time: 1.32 mins

Server is running on http://pr902.butyr.org/templates/basic/
Run the Test Suite: http://pr902.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/3b311ab37a5754d/output.txt

Total script time: 1.54 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e013279c69e988e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/e013279c69e988e/output.txt

Total script time: 1.93 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/a851b1581aef99c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/a851b1581aef99c/output.txt

Total script time: 2.00 mins

**Pull request merged**